routes/store.php

```diff
@@ -141,36 +141,46 @@
     ->middleware('can:view,registration');
 
 
-    // Printables
+    // Printables/Downloadables
 
     // Print a specific Family Form for User Centre (Edit page)
     Route::get('/registrations/{registration}/print', [
         'as' => 'store.registration.print',
         'uses' => 'RegistrationController@printOneIndividualFamilyForm',
-    ]);
+    ])
+        // Can view, can print individual form.
+        ->middleware('can:view,registration');
 
     // Batch print Family Forms for User Centre
     Route::get('/registrations/print', [
         'as' => 'store.registrations.print',
         'uses' => 'RegistrationController@printBatchIndividualFamilyForms',
     ]);
+    // If we keep this it needs a guard.
```

Comment:
Which thing does the comment refer to?

Reply:
The batch print famly route; Turns out another card in this sprint will remove it.